test/secret/secret_s1_test.go

```diff
@@ -123,7 +123,7 @@ var _ = Describe("Secret Test for SVA S1", func() {
 		})
 	})
 
-	Context("Standalone deployment (S1) with LM", func() {
+	XContext("Standalone deployment (S1) with LM", func() {
```

Comment:
Why are we disabling this?

Reply:
Accidental test code....... Reverting it.